Zend/tests/func_get_arg_stub.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test func_get_args func info correctness
+--FILE--
+<?php
+
+(function () {
+    var_dump(func_get_args(null));
```

Comment:
While we're here, can you please add a zend_parse_parameters_none() call to the function? This should be throwing.